server/server_signals.go

```diff
@@ -0,0 +1,38 @@
+// +build !windows
+
+package server
+
+import (
+	"os/signal"
+	"syscall"
+
+	"github.com/containous/traefik/log"
+)
+
+func (server *Server) configureSignals() {
+	signal.Notify(server.signals, syscall.SIGINT, syscall.SIGTERM, syscall.SIGUSR1)
```

Comment:
What happens if we try to register `SIGUSR1` on Windows? Will the machine burn up in flames, or may the signal rather just get transmitted?

In case of the latter, could we maybe drop `server_signals_windows.go`?